CHANGELOG.md

```diff
@@ -8,87 +9,92 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-* [#29](https://github.com/kaklakariada/fritzbox-java-api/issues/29) Add support for login via Pbkdf2 challenge-response (propsed by [@linnex81](https://github.com/linnex81))
+- [#29](https://github.com/kaklakariada/fritzbox-java-api/issues/29) Add support for login via Pbkdf2 challenge-response (propsed by [@linnex81](https://github.com/linnex81))
 
 ### Fixed
 
-* [#37](https://github.com/kaklakariada/fritzbox-java-api/issues/37) Support empty device lists
+- [#37](https://github.com/kaklakariada/fritzbox-java-api/issues/37) Support empty device lists
 
 ### Refactoring
 
-* [#50](https://github.com/kaklakariada/fritzbox-java-api/pull/50) Migrate to Junit 5
+- [#50](https://github.com/kaklakariada/fritzbox-java-api/pull/50) Migrate to Junit 5
+
+## [1.6.1] - unreleased
+
+- datatime attribute added in devicestats XML response since Fritz OS 7.50
```

Comment:
```suggestion
- [#58](https://github.com/kaklakariada/fritzbox-java-api/pull/58) datatime attribute added in devicestats XML response since Fritz OS 7.50
```

Reply:
I've converted the return value into an Instant as you suggested and added a few small tests into your existing test classes. 
Furthermore I've merged with your newest commit on master, hope you are fine with it.

In case I should add anything, let me know.